encryption/c#-example/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text; 
+
+
+namespace Encryption
+{
+    class Program
+    {
+         
+        public static void Main(string[] args)
+        {
+          
+            string plaintext = "Hello World";
+            string key = "abcd123456789";

```

Comment:
for the key, is there any specific requirement from the IT Sec? maybe the key must contain a special character, capital letter, etc. If so please add the comment as well yah @brigitacd